### PR TITLE
Test Modules: Fix module-base-full base images to specify moduleuser in dockerfile

### PR DESCRIPTION
**Pull request by and-rewsmith**
The base images are missing moduleuser. Adding to error observed in longhaul

### CODE REVIEW

File: test/connectivity/modules/NetworkController/docker/linux/arm32v7/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-ARG base_tag=1.0.6.0-linux-arm32v7
+ARG base_tag=1.0.6.1-linux-arm32v7
```
Comment (yophilav) [inline test/connectivity/modules/NetworkController/docker/linux/arm32v7/Dockerfile:1]: What's the reason why {NetworkController, TestAnalyzer, TestResultCoordinator, TwinTesters} are the only one needs this non-root user? (except edgeAgent which already has it).

Comment (and-rewsmith) [reply test/connectivity/modules/NetworkController/docker/linux/arm32v7/Dockerfile:1]: iotedge service needs moduleuser or edgehubuser to start the module. Every base image has it. Module base used by many more for example:
https://github.com/Azure/iotedge/blob/master/edge-modules/SimulatedTemperatureSensor/docker/linux/arm32v7/base/Dockerfile

**Comment by yophilav** [comment]
Please only merge when build image pipeline successfully build. Thank you!

**Comment by and-rewsmith** [comment]
Closing as this base branch is based on iiot

**Pull request closed without merging**